code/controllers/subsystem/statpanel.dm

```diff
@@ -395,11 +395,62 @@ SUBSYSTEM_DEF(statpanels)
 	listed_turf = new_turf
 	if(!client)
 		return
-	if(!client.obj_window)
-		client.obj_window = new(client)
-	if(listed_turf)
-		client.stat_panel.send_message("create_listedturf", listed_turf.name)
-		client.obj_window.start_turf_tracking()
-	else
-		client.stat_panel.send_message("remove_listedturf")
-		client.obj_window.stop_turf_tracking()
+
+	var/datum/tgui/ui = SStgui.try_update_ui(usr, src, "LootPanel")
+	if(!ui)
+		ui = new(usr, src, "LootPanel")
+		ui.open()
+
+/mob/ui_data(mob/user)
```

Comment:
I don't think anything should ever be doing something like this, please make it a separate datum

Reply:
As of right now it can't detect that I've moved tiles, so yes lol.
However: window opens